web-api/src/persistence/postgres/cases/getEligibleCasesForTrialCity.ts

```diff
@@ -57,44 +60,46 @@ export const getEligibleCasesForTrialCity = async ({
   // use batchGet instead of queries with Promise.all?
   const casePromises = dbCases.map(async c => {
     const [privatePractitioners, irsPractitioners] = await Promise.all([
-      query({
-        ExpressionAttributeNames: {
-          '#pk': 'pk',
-          '#sk': 'sk',
-        },
-        ExpressionAttributeValues: {
-          ':pk': `case|${c.docketNumber}`,
-          ':skPrefix': 'privatePractitioner|',
-        },
-        KeyConditionExpression: '#pk = :pk AND begins_with(#sk, :skPrefix)',
+      getPrivatePractitionersOnCase({
+        docketNumber: c.docketNumber,
         applicationContext,
       }),
-      query({
-        ExpressionAttributeNames: {
-          '#pk': 'pk',
-          '#sk': 'sk',
-        },
-        ExpressionAttributeValues: {
-          ':pk': `case|${c.docketNumber}`,
-          ':skPrefix': 'irsPractitioner|',
-        },
-        KeyConditionExpression: '#pk = :pk AND begins_with(#sk, :skPrefix)',
+      getIrsPractitionersOnCase({
+        docketNumber: c.docketNumber,
         applicationContext,
       }),
     ]);
 
-    return purgeDynamoKeys({
+    const dynamoData = purgeDynamoKeys<
+      any,
+      {
+        irsPractitioners: IrsPractitioner[];
+        privatePractitioners: PrivatePractitioner[];
+      }
```

Comment:
Basically saying, "look, I know you _think_ these are each `TDynamoRecord[]` ... but I promise, they aren't!"